code-challenges.test.js

```diff
@@ -27,41 +27,171 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+//         expect(capFirstLett(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion",     "Nephew"])  
+//     })
+// })
+
+// FAIL  ./code-challenges.test.js
+// capFirstLett
+//   ✕ returns an array with all the words capitalized (1 ms)
+// ● capFirstLett › returns an array with all the words capitalized
+//   ReferenceError: capFirstLett is not defined
+// GOOD FAIL  
+// PASS  ./code-challenges.test.js
+//   capFirstLett
+//     ✓ returns an array with all the words capitalized (2 ms)
+
+
 // b) Create the function that makes the test pass.
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+
+// const capFirstLett = (array) => {
+//     return array.map((value) => {
+//         return value[0].toUpperCase() + value.substring(1)
+    
+//     })
+// } 
+
+// console.log(capFirstLett(randomNouns1))
+// console.log(capFirstLett(randomNouns2))
 
 // Pseudo code:
+// Function: capFirstLett
+// Input: randomNouns1 and randomNouns2
+// Output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+//         ["Temperature", "Database", "Chopsticks", "Mango"]
+// Process: .map() AND .toUpperCase() + .substring()
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
+// const vowelTester1 = "learn"
 // Expected output: 1
-const vowelTester2 = "academy"
+// const vowelTester2 = "academy"
 // Expected output: 0
-const vowelTester3 = "challenges"
+// const vowelTester3 = "challenges"
 // Expected output: 2
 
+// Input: "learn"
+// output: 1
+// Input: "academy"
+// Output: 0
+// Input: "challenges"
+// Output: 2
+
+// describe("firstVowel", () => {
+//     it("the index of the first vowel", () => {
+//         const vowelTester1 = "learn"
+//         const vowelTester2 = "academy"
+//         const vowelTester3 = "challenges"
+//         expect(firstVowel(vowelTester1)).toEqual(1)
+//         expect(firstVowel(vowelTester2)).toEqual(0)
+//         expect(firstVowel(vowelTester3)).toEqual(2)
+//     })
+// })
+// FAIL  ./code-challenges.test.js
+// firstVowel
+//   ✕ the index of the first vowel
+// ● firstVowel › the index of the first vowel
+//   ReferenceError: firstVowel is not defined
 // b) Create the function that makes the test pass.
+// Create a function that takes in a string and logs the index of the first vowel.
+    //     const vowelTester1 = "learn"
+    //     const vowelTester2 = "academy"
+    //     const vowelTester3 = "challenges"
+
+    //    const firstVowel = (string) => {
+    //     let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
+    //     return firstVowel.indexOf().filter(
```

Comment:
Good approach holding your vowels in an array. One way you could continue this approach would be to start an iteration and check if the current letter in the given string is included in the array. If it's included, that means the current letter is a vowel, and then you would want to return that index.